wordpress-mobile/WordPress-iOS
# Gutenberg: Update to v1.17.0

This PR updates Gutenberg to v1.15.0
`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1566

To test:

Please copy the following checklist into a new comment and mark each test with ✅, ⚠️ or ❌ based on your test result. Check the individual PR for testing steps.

⏳ Activate Preformatted block https://github.com/wordpress-mobile/gutenberg-mobile/pull/1442 
⏳ Send backspace event to JS in more scenarios https://github.com/wordpress-mobile/gutenberg-mobile/pull/1517
⏳ Generate i18n files for WPAndroid and WPiOS to use https://github.com/wordpress-mobile/gutenberg-mobile/pull/1520
⏳ `shouldInteractWithURL` will return always false to avoid crashes https://github.com/wordpress-mobile/gutenberg-mobile/pull/1523
⏳ Include the block title in unsupported blocks https://github.com/wordpress-mobile/gutenberg-mobile/pull/1533
⏳ Support copy of url in clipboard. https://github.com/wordpress-mobile/gutenberg-mobile/pull/1537
⏳ Remove custom paragraph options for media-text https://github.com/wordpress-mobile/gutenberg-mobile/pull/1538
⏳ Implement client-defined media options. https://github.com/wordpress-mobile/gutenberg-mobile/pull/1539
⏳ Show new block indicator in default appender too https://github.com/wordpress-mobile/gutenberg-mobile/pull/1542
⏳ Media & Text alignment options https://github.com/wordpress-mobile/gutenberg-mobile/pull/1546
⏳ Prevents selection after ZWSP character https://github.com/wordpress-mobile/gutenberg-mobile/pull/1552
⏳ Fix paste in title https://github.com/wordpress-mobile/gutenberg-mobile/pull/1575

Update release notes:
